backend/Models/Data/User.cs

```diff
@@ -2,4 +2,15 @@
 
 namespace WhaleSpotting.Models.Data;
 
-public class User : IdentityUser<int> { }
+public class User : IdentityUser<int>
+{
+    public string? FirstName { get; set; }
+
+    public string? LastName { get; set; }
+
+    public int TotalPointsEarned { get; set; }
```

Comment:
I think this should default to 0 when a user is being created - please can you add that?